packages/sdks-tests/src/e2e-tests/text-eval.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from '@playwright/test';
+import { test } from '../helpers/index.js';
+
+test.describe('Text Eval', () => {
+  test('binds text to state', async ({ page }) => {
+    await page.goto('/text-eval');
+    await expect(page.getByText('Device Size: large')).toBeVisible();
```

Comment:
Something is off about this test.

React Native has a manual override for the Text block component which still doesn't handle bindings:

![CleanShot 2024-08-19 at 11 55 56@2x](https://github.com/user-attachments/assets/8e4feb9c-214a-4eee-9d2b-3c1ebb695a4e)

For some reason playwright still passes the test for it...not sure why.


My main point is that this feature is still unsupported in that SDK. It needs to be added to this file: https://github.com/BuilderIO/builder/blob/main/packages/sdks/overrides/react-native/src/blocks/text/text.jsx#L90-L105

The other issue is understanding why this test is passing despite the `large` text not being in the DOM...

Reply:
made these changes. couldnt recreate locally the phantom test passing. modified the test to explicitly fail and it behaved as expected, and after my changes the test also passes correctly as well.